database-commons/src/main/java/io/cdap/plugin/db/DBRecord.java

```diff
@@ -364,16 +363,6 @@ protected void writeBytes(PreparedStatement stmt, int fieldIndex, int sqlIndex,
     stmt.setBytes(sqlIndex, byteValue);
   }
 
-  protected void writeInt(PreparedStatement stmt, int fieldIndex, int sqlIndex, Object fieldValue) throws SQLException {
```

Comment:
why are we removing this? Does the driver do the right thing if we do setInt() for tinyint/smallint columns?

Reply:
setShort does not handle max value for smallint unsigned type. 
Yes, setInt internally does the right thing for tinyint/smallint types, hence intended to clear the logic a bit.